interpreter/package.go

```diff
@@ -60,23 +80,29 @@ func (p *Package) Type() semantic.MonoType {
 }
 func (p *Package) Get(name string) (values.Value, bool) {
 	v, ok := p.object.Get(name)
-	if !ok {
-		return p.options.Get(name)
+	if ok && values.IsOption(v) {
+		// If this value is an option, the option may have
+		// been overriden. Check the override map.
+		if ov, ok := p.options[name]; ok {
+			v = ov
+		}
 	}
-	return v, true
+	return v, ok
 }
 func (p *Package) Set(name string, v values.Value) {
-	p.object.Set(name, v)
+	panic(errors.New(codes.Internal, "package members cannot be modified"))
 }
 func (p *Package) SetOption(name string, v values.Value) {
-	p.options.Set(name, v)
+	if p.options == nil {
+		p.options = make(map[string]values.Value)
+	}
+	p.options[name] = v
 }
 func (p *Package) Len() int {
 	return p.object.Len()
 }
 func (p *Package) Range(f func(name string, v values.Value)) {
 	p.object.Range(f)
-	p.options.Range(f)
```

Comment:
This needs to iterator over the options as well.

Reply:
I changed the package so the options are part of the object. Maybe that was incorrect. It seemed to me like when an option is defined, it is defined as a variable that can also be used in the same context. Instead of being in separate objects, the option is wrapped in a `values.Option` to denote it as something special.

So when you do:

```
option prefix = "influxdata_"
my_ident = (name) => prefix + name
```

The prefix becomes part of the object and can be accessed as if it were a normal variable, but it gets wrapped with `values.Option` so `SetOption` will work on it.

Reply:
I see, does that mean we need to check the `options` map for overrides during this Range function? Basically copy the `Get` logic into the method as well?

Reply:
We do. I seem to have made a mistake here.